Automattic/jetpack
# My Jetpack: Add stats to VideoPress card

Fixes #29702 

## Proposed changes:

* Adds the necessary presentational components for many rich cards
* Applies it to the VideoPress card, adding stats

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Install VideoPress
* Go to My Jetpack
* Check that the stats are loaded in the card

Loading | Positive | No change | Negative | No available stats
-|-|-|-|-
![2023-04-19_14-42-38](https://user-images.githubusercontent.com/8486249/233157263-d4a8dee1-02be-4767-b813-d70557131e1d.png) | ![2023-04-19_14-35-30](https://user-images.githubusercontent.com/8486249/233157285-a83422d3-4d66-4637-9349-3e5b4e03a48f.png) | ![2023-04-19_14-41-06](https://user-images.githubusercontent.com/8486249/233157304-0ae99e38-21b1-407e-b86f-312a86873d52.png) | ![2023-04-19_14-41-30](https://user-images.githubusercontent.com/8486249/233157323-6b81bdb9-5440-41c5-983b-1375b64fac42.png) | ![2023-04-19_14-43-02](https://user-images.githubusercontent.com/8486249/233157330-b9903f86-d05a-41fc-aea0-9bef202ef626.png)
